src/reconcile/rec_col.c

```diff
@@ -64,6 +65,8 @@ __wt_bulk_insert_fix(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk, bool delet
     ++cbulk->entry;
     ++r->recno;
 
+    WT_TIME_WINDOW_INIT(&tw);
+    WT_TIME_AGGREGATE_UPDATE(session, &r->cur_ptr->ta, &tw);
```

Comment:
When you initialize an aggregate for updating it sets the minimums to max and the maximums to min, so the aggregate window only gets narrower as updates are applied. But then, if it doesn't get updated at least once, it has start > stop and fails validation. This is why there are a bunch of places that aggregate in a stable timestamp; it's because things explode if you don't. :-|

Reply:
That deserves a comment (or a new macro).

Reply:
Fixed.